examples/hubert/preprocess.py

```diff
@@ -60,6 +60,12 @@ def _parse_args():
         type=int,
         help="The number of clusters for KMeans clustering.",
     )
+    parser.add_argument(
+        "--percent",
+        default=-1,
+        type=float,
+        help="The number of clusters for KMeans clustering.",
```

Comment:
should this be something like this:

```suggestion
        help="The percent of data for KMeans clustering. If negative, use all data. (Default: -1)",
```